x-pack/legacy/plugins/integrations_manager/public/register_feature.ts

```diff
@@ -4,20 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import {
-  FeatureCatalogueCategory,
-  FeatureCatalogueRegistryProvider,
-} from 'ui/registry/feature_catalogue';
+import { npSetup } from 'ui/new_platform';
+import { FeatureCatalogueCategory } from 'ui/registry/feature_catalogue';
```

Comment:
Is it okay to continue referencing from this module even though it will be deprecated soon ( based on his comments https://github.com/elastic/kibana/pull/48818) ?

Reply:
Yeah, rather use the type from the NP plugin.

Reply:
Updated in a01eea4